lib/services/workflow-api-service.js

```diff
@@ -93,6 +95,7 @@ function workflowApiServiceFactory(
             });
         })
         .then(function(graph) {
+            sanitizer.decrypt(graph);
```

Comment:
so, not really sure I follow... if your worried this will not fix the issue, I have tested it locally.